### PR TITLE
Check style fix

### LINKED ISSUE #932: Fix check style problem with class BasePageRequest
The class `org.apache.shardingsphere.elasticjob.lite.console.dto.request.BasePageRequest` is break rule of check style.
There are use `// CHECKSTYLE:OFF` to avoid checkstyle validate, but protected field modifiers do not allow.

Please remove `// CHECKSTYLE:OFF` and provide protected getter and setter method to instead of protect fields.

### PR DESCRIPTION
**Pull request by taojintianxia**
Fixes #932

### CODE REVIEW

File: elastic-job-lite-console/src/main/java/org/apache/shardingsphere/elasticjob/lite/console/dto/request/FindJobExecutionEventsRequest.java [hunk 1]
```diff
@@ -55,8 +55,8 @@
      * @param pageSize page size
      */
     public FindJobExecutionEventsRequest(final Integer pageSize, final Integer pageNumber) {
-        this.pageSize = pageSize;
-        this.pageNumber = pageNumber;
+        setPageSize(pageSize);
```
Comment (terrymanu) [inline elastic-job-lite-console/src/main/java/org/apache/shardingsphere/elasticjob/lite/console/dto/request/FindJobExecutionEventsRequest.java]: Please just use this.xxx to assign regular fields

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: elastic-job-lite-console/src/main/java/org/apache/shardingsphere/elasticjob/lite/console/dto/request/FindJobExecutionEventsRequest.java (hunk 1 above)
Comment (taojintianxia) [reply elastic-job-lite-console/src/main/java/org/apache/shardingsphere/elasticjob/lite/console/dto/request/FindJobExecutionEventsRequest.java]: done

File: examples/elastic-job-example-lite-java/src/main/java/org/apache/shardingsphere/elasticjob/lite/example/JavaMain.java [hunk 2]
```diff
@@ -55,7 +55,7 @@
     private static final String EVENT_RDB_STORAGE_USERNAME = "sa";
     
     private static final String EVENT_RDB_STORAGE_PASSWORD = "";
-    
+
```
Comment (terrymanu) [inline examples/elastic-job-example-lite-java/src/main/java/org/apache/shardingsphere/elasticjob/lite/example/JavaMain.java]: Please do not change original indent

File: elastic-job-lite-console/src/main/java/org/apache/shardingsphere/elasticjob/lite/console/dto/request/FindJobStatusTraceEventsRequest.java [hunk 3]
```diff
@@ -52,15 +52,11 @@
     private Date end;
     
     public FindJobStatusTraceEventsRequest(final Integer pageSize, final Integer pageNumber) {
-        this.pageSize = pageSize;
-        this.pageNumber = pageNumber;
+        super(pageSize, pageNumber, null, null);
     }
-    
+
```
Comment (terrymanu) [inline elastic-job-lite-console/src/main/java/org/apache/shardingsphere/elasticjob/lite/console/dto/request/FindJobStatusTraceEventsRequest.java]: Please do not change original indent

**Review by terrymanu — CHANGES_REQUESTED** [review]

File: examples/elastic-job-example-lite-java/src/main/java/org/apache/shardingsphere/elasticjob/lite/example/JavaMain.java (hunk 2 above)
Comment (taojintianxia) [reply examples/elastic-job-example-lite-java/src/main/java/org/apache/shardingsphere/elasticjob/lite/example/JavaMain.java]: done

File: elastic-job-lite-console/src/main/java/org/apache/shardingsphere/elasticjob/lite/console/dto/request/FindJobStatusTraceEventsRequest.java (hunk 3 above)
Comment (taojintianxia) [reply elastic-job-lite-console/src/main/java/org/apache/shardingsphere/elasticjob/lite/console/dto/request/FindJobStatusTraceEventsRequest.java]: done

**Review by terrymanu — APPROVED** [review]

**Pull request merged**